mlflow/pyfunc/model.py

```diff
@@ -660,6 +666,13 @@ def predict(self, model_input, params: Optional[dict[str, Any]] = None):
             dict or string. Chunk dict fields are determined by the model implementation.
         """
         parameters = inspect.signature(self.python_model.predict).parameters
+        if invalid_params := set(parameters) - {"context", "model_input", "params"}:
+            warnings.warn(
+                _INVALID_SIGNATURE_ERROR_MSG.format(
+                    func_name="predict", invalid_params=invalid_params
+                ),
```

Comment:
```suggestion
                ),
                FutureWarning,
```

Can we add `FutureWarning`?